Gemfile

```diff
@@ -8,10 +8,6 @@ group :test do
   gem "codeclimate-test-reporter", "~> 1.0.0", :require => false
 end
 
-unless dependencies.detect { |d| d.name == "xclarity_client" }
-  gem "xclarity_client"
-end
-
```

Comment:
I had it there, as I think the Amazon provider used to have it. I'm sure it didn't work without it?